server/database.js

```diff
@@ -0,0 +1,9 @@
+var mysql = require('mysql')
+var connection = mysql.createConnection({
```

Comment:
This is a minor comment, but I would suggest putting the login information for mysql (or at least the password) in a config file that is ignored by git. You could then import that ignored file here.

Alternatively you could set this in environment variables.